benchmark.sh

```diff
@@ -2,5 +2,5 @@ if [ $# -ne 1 ]; then
     echo "[ERROR] Need an argument. Usage: ./benchmark.sh PYTHONPATH"
 else
     make clean
-    PYTHONPATH=$1 make
+    PYTHONPATH=$1 make multitime
```

Comment:
Ah, sorry. I think we dont need this script any more, right? We just run `make multitime` (maybe rename the target to `benchmark`).